src/client.js

```diff
@@ -269,6 +269,113 @@ export default class StitchClient {
       .then(response => response.json());
   }
 
+  /**
+   * Returns an array of api keys
+   *
+   * @returns {Promise}
+   */
+  getApiKeys() {
+    return this._do(
+      '/auth/me/api_keys',
+      'GET',
+      {
+        rootURL: this.rootURLsByAPIVersion[v1][API_TYPE_APP],
+        useRefreshToken: true
+      },
+    )
+      .then((response) => response.json());
+  }
+
+  /**
+   * Creates an user api key
+   *
+   * @param {String} body a JSON object of the name of the new api key
```

Comment:
I'm working on documenting this right now and I think this is a little confusing. It seems like it would be better to just accept the string and build the JSON body in the function here. That's what the other SDKs are doing.

Reply:
Good catch. Less knowledge required for the end user.